cfme/tests/configure/test_access_control.py

```diff
@@ -2888,3 +2889,38 @@ def test_default_tenat_name_rail_console():
             2. tenant name should be updated on rail console
     """
     pass
+
+
+@pytest.mark.meta(blockers=[BZ(1803952, forced_streams=["5.10"])])
+@test_requirements.rbac
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1803952])
+def test_select_edit_group(appliance):
+    """
+    Should be able to see "edit group" option for existing group after selecting the group
+    also check error in production.log log file
+    Bugzilla:
+        1803952
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Configuration
+        caseimportance: critical
+        initialEstimate: 1/4h
+        testSteps:
+            1. Go to Configuration--> Acesss controls
+            2. Select existing custom group and Configure
+        expectedResults:
+            1.
+            2. Able to see Edit Group option
+    """
+    view = navigate_to(appliance.collections.groups, "All")
+    msg = ".*FATAL.*Error caught: [NoMethodError].*"
+    with LogValidator("/var/www/miq/vmdb/log/production.log",
+                      failure_patterns=[msg]).waiting(timeout=120):
+
+        # Selecting first row from all group
+        row = view.table[0]
```

Comment:
The docstring description and the testSteps and expectedResults sections indicate that the test should be for a custom group, and that the option is called 'Edit Group'. But the test clicks on the first group in the list of all groups, which is probably going to be the default group EvmGroup-administrator, not a custom group, and the option is 'Edit the selected Group'.

Reply:
Done

Reply:
The docstring description and expectedResults are still wrong:
```
Should be able to see "edit group" option for existing group after selecting the group
```
and
```
2. Able to see Edit Group option
```
The option is `Edit the selected Group`, not `edit group` or `Edit Group`.